eosdis-nasa/earthdata-pub-dashboard
# Hotfix/split conversation component

# Description

This is an intermediate step in trying to modularize parts of the conversation component to allow easier maintenance in the future

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Create a new request.
4. Test adding a new note in the request conversation.
5. Testing limiting visibility while adding a new note in the request conversation.
